src/ert/enkf_main.py

```diff
@@ -162,6 +159,29 @@ def _seed_sequence(seed: Optional[int]) -> int:
     return int_seed
 
 
+def save_design_matrix_to_ensemble(
```

Comment:
Should this be a method in the ensemble class? `my_ensemble.save_design_matrix(...)`

Reply:
There is already `ensemble.save_parameters`, which represents the interface for storing parameters. Not sure if it would bring any value having it there as it is a special "class" of parameters.